float32[] joint_target_positions